src/components/DigitizeProject.vue

```diff
@@ -86,6 +93,10 @@ export default defineComponent({
         this.updateTaskFeature()
       }
     },
+    createInformationPages,
+    createFallbackInformationPages() {
+      return undefined
```

Comment:
same as above: why is this undefined?

Reply:
Added a comment here as well. Tutorials for Digitize projects do not exist at the moment. Fallback information pages can better be discussed at the time we implement tutorials for this project type.